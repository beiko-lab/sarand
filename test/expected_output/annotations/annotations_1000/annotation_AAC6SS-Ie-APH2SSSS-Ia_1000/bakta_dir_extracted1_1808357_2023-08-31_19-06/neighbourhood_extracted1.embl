ID   contig_1; ; linear; DNA; ; UNC; 3440 BP.
XX
AC   contig_1;
XX
DE   contig_1, whole genome shotgun sequence
XX
OS   .
OC   .
XX
CC   Annotated with Bakta
CC   Software: v1.8.1
CC   Database: v5.0, light
CC   DOI: 10.1099/mgen.0.000685
CC   URL: github.com/oschwengers/bakta
CC   
CC   ##Genome Annotation Summary:##
CC   Annotation Date                :: 08/31/2023, 19:06:17
CC   CDSs                           ::     2
CC   tRNAs                          ::     0
CC   tmRNAs                         ::     0
CC   rRNAs                          ::     0
CC   ncRNAs                         ::     0
CC   regulatory ncRNAs              ::     0
CC   CRISPR Arrays                  ::     0
CC   oriCs/oriVs                    ::     0
CC   oriTs                          ::     0
CC   gaps                           ::     0
CC   pseudogenes                    ::     0
XX
FH   Key             Location/Qualifiers
FH
FT   source          1..3440
FT                   /mol_type="genomic DNA"
FT   gene            554..1000
FT                   /locus_tag="FKBKAJ_00005"
FT   CDS             554..1000
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_Q7WTB9"
FT                   /product="N-acetyltransferase domain-containing protein"
FT                   /locus_tag="FKBKAJ_00005"
FT                   /protein_id="gnl|Bakta|FKBKAJ_00005"
FT                   /translation="MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSF
FT                   GFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEI
FT                   GCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_Q7WTB9"
FT   gene            1001..2440
FT                   /locus_tag="FKBKAJ_00010"
FT                   /gene="aac(6')-Ie"
FT   CDS             1001..2440
FT                   /db_xref="NCBIProtein:WP_001028144.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P0A0C2"
FT                   /product="bifunctional aminoglycoside N-acetyltransferase
FT                   AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia"
FT                   /locus_tag="FKBKAJ_00010"
FT                   /protein_id="gnl|Bakta|FKBKAJ_00010"
FT                   /translation="MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLES
FT                   LKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGE
FT                   PNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHE
FT                   LHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNE
FT                   YIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTF
FT                   LTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIY
FT                   NDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGI
FT                   IDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIK
FT                   NIKQEFIENGRKEIYKRTYKD"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P0A0C2"
FT                   /gene="aac(6')-Ie"
XX
SQ   Sequence 3440 BP; 1279 A; 404 C; 594 G; 1163 T; 0 other;
     aaaatacact caaaggcatt gaatgtattt acgctctata taaaaagaac cgcagatctc        60
     ttcagatcta cggattttcg ccatgccacg aaattaggga tatgttagcc agttaaggga       120
     acaacgacat gataaaatgg tagttagcta tatttttcta actttgcaac agaaccccta       180
     aaaaaaattc aaaatctttc agagaagctg ttaaaggaat ttttaagttc acagatatta       240
     acttagcgcg tgaggctaaa aatcgattga ttcatgatta tatcgatcaa ccaaaatatt       300
     caaaagcttg cgcatcattg gatgatggat tcgaagacgc ctttcaatat accgtacaag       360
     gaaattccca caatcgacta aagagtacca atctaattga acgactgaat caagaagtac       420
     gcagaagaga aaagattatt cgcatcttcc ccaatcaaac atcagccaat cgcttaattg       480
     gagccgttct tatggaccta catgatgaat ggatttattc ttcaagaaaa tacatcaatt       540
     ttgataagta gaaatggtaa aaacattgta tagcatttta cacaggagtc tggacttgac       600
     tgagtttatg gaagaagttt taattgatga taatatggtt tttgatattg ataatttaaa       660
     aggatttctt aatgatacca gttcatttgg gtttatagct aaagaaaata ataaaattat       720
     aggatttgca tattgctata cacttttaag acctgatgga aaaacaatgt tttatttaca       780
     ctcaatagga atgttaccta actatcaaga caaaggttat ggttcaaaat tattatcttt       840
     tattaaggaa tattctaaag agattggttg ttctgaaatg tttttaataa ctgataaagg       900
     taatcctaga gcttgccatg tatatgaaaa attaggtggt aaaaatgatt ataaagatga       960
     aatagtatat gtatatgatt atgaaaaagg tgataaataa atgaatatag ttgaaaatga      1020
     aatatgtata agaactttaa tagatgatga ttttcctttg atgttaaaat ggttaactga      1080
     tgaaagagta ttagaatttt atggtggtag agataaaaaa tatacattag aatcattaaa      1140
     aaaacattat acagagcctt gggaagatga agtttttaga gtaattattg aatataacaa      1200
     tgttcctatt ggatatggac aaatatataa aatgtatgat gagttatata ctgattatca      1260
     ttatccaaaa actgatgaga tagtctatgg tatggatcaa tttataggag agccaaatta      1320
     ttggagtaaa ggaattggta caagatatat taaattgatt tttgaatttt tgaaaaaaga      1380
     aagaaatgct aatgcagtta ttttagaccc tcataaaaat aatccaagag caataagggc      1440
     ataccaaaaa tctggtttta gaattattga agatttgcca gaacatgaat tacacgaggg      1500
     caaaaaagaa gattgttatt taatggaata tagatatgat gataatgcca caaatgttaa      1560
     ggcaatgaaa tatttaattg agcattactt tgataatttc aaagtagata gtattgaaat      1620
     aatcggtagt ggttatgata gtgtggcata tttagttaat aatgaataca tttttaaaac      1680
     aaaatttagt actaataaga aaaaaggtta tgcaaaagaa aaagcaatat ataatttttt      1740
     aaatacaaat ttagaaacta atgtaaaaat tcctaatatt gaatattcgt atattagtga      1800
     tgaattatct atactaggtt ataaagaaat taaaggaact tttttaacac cagaaattta      1860
     ttctactatg tcagaagaag aacaaaattt gttaaaacga gatattgcca gttttttaag      1920
     acaaatgcac ggtttagatt atacagatat tagtgaatgt actattgata ataaacaaaa      1980
     tgtattagaa gagtatatat tgttgcgtga aactatttat aatgatttaa ctgatataga      2040
     aaaagattat atagaaagtt ttatggaaag actaaatgca acaacagttt ttgagggtaa      2100
     aaagtgttta tgccataatg attttagttg taatcatcta ttgttagatg gcaataatag      2160
     attaactgga ataattgatt ttggagattc tggaattata gatgaatatt gtgattttat      2220
     atacttactt gaagatagtg aagaagaaat aggaacaaat tttggagaag atatattaag      2280
     aatgtatgga aatatagata ttgagaaagc aaaagaatat caagatatag ttgaagaata      2340
     ttatcctatt gaaactattg tttatggaat taaaaatatt aaacaggaat ttatcgaaaa      2400
     tggtagaaaa gaaatttata aaaggactta taaagattga ttatataata tatgaaaagc      2460
     tattataaaa gacattagta ttaaatagtt taaaaaaatg aaaaataata aaggaagtga      2520
     gtcaagtcca gactcctgtg taaaatgcta tacaatgttt ttaccatttc tacttatcaa      2580
     aattgatgta ttttcttgaa gaataaatcc attcatcatg taggtccata agaacggctc      2640
     caattaagcg attggctgat gtttgattgg ggaagatgcg aataatcttt tctcttctgc      2700
     gtacttcttg attcagtcgt tcaattagat tggtactctt tagtcgattg tgggaatttc      2760
     cttgtacggt atattgaaag gcgtcttcga atccatcatc caatgatgcg caagcttttg      2820
     aatattttgg ttgatcgata taatcatgaa tcaatcgatt tttagcctca cgcgctaagt      2880
     taatatctgt gaacttaaaa attcctttaa cagcttctct gaaagatttt gaattttttt      2940
     taggggttct gttgcaaagt tagaaaaata tagctaacta ccattttatc atgtcgttgt      3000
     tcccttaact ggctaacata tccctaattt cgtggcatgg cgaaaatccg tagatctgaa      3060
     gagatctgcg gttcttttta tatagagcgt aaatacattc aatgcctttg agtgtatttt      3120
     ttgccgtatt gatactttga tatcttgtct ttcttacttt aatatgacgg tgatcttgct      3180
     caatgaggtt attcagatat ttcgatgtac aatgacagtc aggtttaagt ttaaaagctt      3240
     taattacttt agctattgct actttcgttg aaggtgcctg atctgtaatt accttttgag      3300
     gtttaccaaa ttgtttaatg agacgtttga taaacgcata tgctgaatga ttatctcgtt      3360
     gcttacgcaa ccaaatatct aatgtatgtc cctctgcatc aatggcacga tataaatagc      3420
     tccattttcc ttttattttg                                                  3440
//
