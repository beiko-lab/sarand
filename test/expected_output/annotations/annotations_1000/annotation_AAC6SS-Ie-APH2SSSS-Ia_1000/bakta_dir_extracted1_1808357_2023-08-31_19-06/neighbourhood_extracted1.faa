>FKBKAJ_00005 N-acetyltransferase domain-containing protein
MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSFGFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEIGCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK
>FKBKAJ_00010 bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia
MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLESLKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGEPNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHELHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNEYIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTFLTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIYNDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGIIDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIKNIKQEFIENGRKEIYKRTYKD
