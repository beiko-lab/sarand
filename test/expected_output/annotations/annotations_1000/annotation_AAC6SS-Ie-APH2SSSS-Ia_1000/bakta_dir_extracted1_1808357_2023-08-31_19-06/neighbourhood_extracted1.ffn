>FKBKAJ_00005 N-acetyltransferase domain-containing protein
ATGGTAAAAACATTGTATAGCATTTTACACAGGAGTCTGGACTTGACTGAGTTTATGGAAGAAGTTTTAATTGATGATAATATGGTTTTTGATATTGATAATTTAAAAGGATTTCTTAATGATACCAGTTCATTTGGGTTTATAGCTAAAGAAAATAATAAAATTATAGGATTTGCATATTGCTATACACTTTTAAGACCTGATGGAAAAACAATGTTTTATTTACACTCAATAGGAATGTTACCTAACTATCAAGACAAAGGTTATGGTTCAAAATTATTATCTTTTATTAAGGAATATTCTAAAGAGATTGGTTGTTCTGAAATGTTTTTAATAACTGATAAAGGTAATCCTAGAGCTTGCCATGTATATGAAAAATTAGGTGGTAAAAATGATTATAAAGATGAAATAGTATATGTATATGATTATGAAAAAGGTGATAAATAA
>FKBKAJ_00010 bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia
ATGAATATAGTTGAAAATGAAATATGTATAAGAACTTTAATAGATGATGATTTTCCTTTGATGTTAAAATGGTTAACTGATGAAAGAGTATTAGAATTTTATGGTGGTAGAGATAAAAAATATACATTAGAATCATTAAAAAAACATTATACAGAGCCTTGGGAAGATGAAGTTTTTAGAGTAATTATTGAATATAACAATGTTCCTATTGGATATGGACAAATATATAAAATGTATGATGAGTTATATACTGATTATCATTATCCAAAAACTGATGAGATAGTCTATGGTATGGATCAATTTATAGGAGAGCCAAATTATTGGAGTAAAGGAATTGGTACAAGATATATTAAATTGATTTTTGAATTTTTGAAAAAAGAAAGAAATGCTAATGCAGTTATTTTAGACCCTCATAAAAATAATCCAAGAGCAATAAGGGCATACCAAAAATCTGGTTTTAGAATTATTGAAGATTTGCCAGAACATGAATTACACGAGGGCAAAAAAGAAGATTGTTATTTAATGGAATATAGATATGATGATAATGCCACAAATGTTAAGGCAATGAAATATTTAATTGAGCATTACTTTGATAATTTCAAAGTAGATAGTATTGAAATAATCGGTAGTGGTTATGATAGTGTGGCATATTTAGTTAATAATGAATACATTTTTAAAACAAAATTTAGTACTAATAAGAAAAAAGGTTATGCAAAAGAAAAAGCAATATATAATTTTTTAAATACAAATTTAGAAACTAATGTAAAAATTCCTAATATTGAATATTCGTATATTAGTGATGAATTATCTATACTAGGTTATAAAGAAATTAAAGGAACTTTTTTAACACCAGAAATTTATTCTACTATGTCAGAAGAAGAACAAAATTTGTTAAAACGAGATATTGCCAGTTTTTTAAGACAAATGCACGGTTTAGATTATACAGATATTAGTGAATGTACTATTGATAATAAACAAAATGTATTAGAAGAGTATATATTGTTGCGTGAAACTATTTATAATGATTTAACTGATATAGAAAAAGATTATATAGAAAGTTTTATGGAAAGACTAAATGCAACAACAGTTTTTGAGGGTAAAAAGTGTTTATGCCATAATGATTTTAGTTGTAATCATCTATTGTTAGATGGCAATAATAGATTAACTGGAATAATTGATTTTGGAGATTCTGGAATTATAGATGAATATTGTGATTTTATATACTTACTTGAAGATAGTGAAGAAGAAATAGGAACAAATTTTGGAGAAGATATATTAAGAATGTATGGAAATATAGATATTGAGAAAGCAAAAGAATATCAAGATATAGTTGAAGAATATTATCCTATTGAAACTATTGTTTATGGAATTAAAAATATTAAACAGGAATTTATCGAAAATGGTAGAAAAGAAATTTATAAAAGGACTTATAAAGATTGA
