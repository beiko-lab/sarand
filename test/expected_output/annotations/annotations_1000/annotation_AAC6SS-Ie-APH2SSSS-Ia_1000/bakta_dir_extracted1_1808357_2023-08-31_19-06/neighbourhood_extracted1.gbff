LOCUS       contig_1                3440 bp    DNA     linear   UNK 31-AUG-2023
DEFINITION  contig_1, whole genome shotgun sequence.
ACCESSION   contig_1
VERSION     contig_1
KEYWORDS    .
SOURCE      None
  ORGANISM  .
            .
COMMENT     Annotated with Bakta
            Software: v1.8.1
            Database: v5.0, light
            DOI: 10.1099/mgen.0.000685
            URL: github.com/oschwengers/bakta
            
            ##Genome Annotation Summary:##
            Annotation Date                :: 08/31/2023, 19:06:17
            CDSs                           ::     2
            tRNAs                          ::     0
            tmRNAs                         ::     0
            rRNAs                          ::     0
            ncRNAs                         ::     0
            regulatory ncRNAs              ::     0
            CRISPR Arrays                  ::     0
            oriCs/oriVs                    ::     0
            oriTs                          ::     0
            gaps                           ::     0
            pseudogenes                    ::     0
FEATURES             Location/Qualifiers
     source          1..3440
                     /mol_type="genomic DNA"
     gene            554..1000
                     /locus_tag="FKBKAJ_00005"
     CDS             554..1000
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_Q7WTB9"
                     /product="N-acetyltransferase domain-containing protein"
                     /locus_tag="FKBKAJ_00005"
                     /protein_id="gnl|Bakta|FKBKAJ_00005"
                     /translation="MVKTLYSILHRSLDLTEFMEEVLIDDNMVFDIDNLKGFLNDTSSF
                     GFIAKENNKIIGFAYCYTLLRPDGKTMFYLHSIGMLPNYQDKGYGSKLLSFIKEYSKEI
                     GCSEMFLITDKGNPRACHVYEKLGGKNDYKDEIVYVYDYEKGDK"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_Q7WTB9"
     gene            1001..2440
                     /locus_tag="FKBKAJ_00010"
                     /gene="aac(6')-Ie"
     CDS             1001..2440
                     /db_xref="NCBIProtein:WP_001028144.1"
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_P0A0C2"
                     /product="bifunctional aminoglycoside N-acetyltransferase
                     AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia"
                     /locus_tag="FKBKAJ_00010"
                     /protein_id="gnl|Bakta|FKBKAJ_00010"
                     /translation="MNIVENEICIRTLIDDDFPLMLKWLTDERVLEFYGGRDKKYTLES
                     LKKHYTEPWEDEVFRVIIEYNNVPIGYGQIYKMYDELYTDYHYPKTDEIVYGMDQFIGE
                     PNYWSKGIGTRYIKLIFEFLKKERNANAVILDPHKNNPRAIRAYQKSGFRIIEDLPEHE
                     LHEGKKEDCYLMEYRYDDNATNVKAMKYLIEHYFDNFKVDSIEIIGSGYDSVAYLVNNE
                     YIFKTKFSTNKKKGYAKEKAIYNFLNTNLETNVKIPNIEYSYISDELSILGYKEIKGTF
                     LTPEIYSTMSEEEQNLLKRDIASFLRQMHGLDYTDISECTIDNKQNVLEEYILLRETIY
                     NDLTDIEKDYIESFMERLNATTVFEGKKCLCHNDFSCNHLLLDGNNRLTGIIDFGDSGI
                     IDEYCDFIYLLEDSEEEIGTNFGEDILRMYGNIDIEKAKEYQDIVEEYYPIETIVYGIK
                     NIKQEFIENGRKEIYKRTYKD"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_P0A0C2"
                     /gene="aac(6')-Ie"
ORIGIN
        1 aaaatacact caaaggcatt gaatgtattt acgctctata taaaaagaac cgcagatctc
       61 ttcagatcta cggattttcg ccatgccacg aaattaggga tatgttagcc agttaaggga
      121 acaacgacat gataaaatgg tagttagcta tatttttcta actttgcaac agaaccccta
      181 aaaaaaattc aaaatctttc agagaagctg ttaaaggaat ttttaagttc acagatatta
      241 acttagcgcg tgaggctaaa aatcgattga ttcatgatta tatcgatcaa ccaaaatatt
      301 caaaagcttg cgcatcattg gatgatggat tcgaagacgc ctttcaatat accgtacaag
      361 gaaattccca caatcgacta aagagtacca atctaattga acgactgaat caagaagtac
      421 gcagaagaga aaagattatt cgcatcttcc ccaatcaaac atcagccaat cgcttaattg
      481 gagccgttct tatggaccta catgatgaat ggatttattc ttcaagaaaa tacatcaatt
      541 ttgataagta gaaatggtaa aaacattgta tagcatttta cacaggagtc tggacttgac
      601 tgagtttatg gaagaagttt taattgatga taatatggtt tttgatattg ataatttaaa
      661 aggatttctt aatgatacca gttcatttgg gtttatagct aaagaaaata ataaaattat
      721 aggatttgca tattgctata cacttttaag acctgatgga aaaacaatgt tttatttaca
      781 ctcaatagga atgttaccta actatcaaga caaaggttat ggttcaaaat tattatcttt
      841 tattaaggaa tattctaaag agattggttg ttctgaaatg tttttaataa ctgataaagg
      901 taatcctaga gcttgccatg tatatgaaaa attaggtggt aaaaatgatt ataaagatga
      961 aatagtatat gtatatgatt atgaaaaagg tgataaataa atgaatatag ttgaaaatga
     1021 aatatgtata agaactttaa tagatgatga ttttcctttg atgttaaaat ggttaactga
     1081 tgaaagagta ttagaatttt atggtggtag agataaaaaa tatacattag aatcattaaa
     1141 aaaacattat acagagcctt gggaagatga agtttttaga gtaattattg aatataacaa
     1201 tgttcctatt ggatatggac aaatatataa aatgtatgat gagttatata ctgattatca
     1261 ttatccaaaa actgatgaga tagtctatgg tatggatcaa tttataggag agccaaatta
     1321 ttggagtaaa ggaattggta caagatatat taaattgatt tttgaatttt tgaaaaaaga
     1381 aagaaatgct aatgcagtta ttttagaccc tcataaaaat aatccaagag caataagggc
     1441 ataccaaaaa tctggtttta gaattattga agatttgcca gaacatgaat tacacgaggg
     1501 caaaaaagaa gattgttatt taatggaata tagatatgat gataatgcca caaatgttaa
     1561 ggcaatgaaa tatttaattg agcattactt tgataatttc aaagtagata gtattgaaat
     1621 aatcggtagt ggttatgata gtgtggcata tttagttaat aatgaataca tttttaaaac
     1681 aaaatttagt actaataaga aaaaaggtta tgcaaaagaa aaagcaatat ataatttttt
     1741 aaatacaaat ttagaaacta atgtaaaaat tcctaatatt gaatattcgt atattagtga
     1801 tgaattatct atactaggtt ataaagaaat taaaggaact tttttaacac cagaaattta
     1861 ttctactatg tcagaagaag aacaaaattt gttaaaacga gatattgcca gttttttaag
     1921 acaaatgcac ggtttagatt atacagatat tagtgaatgt actattgata ataaacaaaa
     1981 tgtattagaa gagtatatat tgttgcgtga aactatttat aatgatttaa ctgatataga
     2041 aaaagattat atagaaagtt ttatggaaag actaaatgca acaacagttt ttgagggtaa
     2101 aaagtgttta tgccataatg attttagttg taatcatcta ttgttagatg gcaataatag
     2161 attaactgga ataattgatt ttggagattc tggaattata gatgaatatt gtgattttat
     2221 atacttactt gaagatagtg aagaagaaat aggaacaaat tttggagaag atatattaag
     2281 aatgtatgga aatatagata ttgagaaagc aaaagaatat caagatatag ttgaagaata
     2341 ttatcctatt gaaactattg tttatggaat taaaaatatt aaacaggaat ttatcgaaaa
     2401 tggtagaaaa gaaatttata aaaggactta taaagattga ttatataata tatgaaaagc
     2461 tattataaaa gacattagta ttaaatagtt taaaaaaatg aaaaataata aaggaagtga
     2521 gtcaagtcca gactcctgtg taaaatgcta tacaatgttt ttaccatttc tacttatcaa
     2581 aattgatgta ttttcttgaa gaataaatcc attcatcatg taggtccata agaacggctc
     2641 caattaagcg attggctgat gtttgattgg ggaagatgcg aataatcttt tctcttctgc
     2701 gtacttcttg attcagtcgt tcaattagat tggtactctt tagtcgattg tgggaatttc
     2761 cttgtacggt atattgaaag gcgtcttcga atccatcatc caatgatgcg caagcttttg
     2821 aatattttgg ttgatcgata taatcatgaa tcaatcgatt tttagcctca cgcgctaagt
     2881 taatatctgt gaacttaaaa attcctttaa cagcttctct gaaagatttt gaattttttt
     2941 taggggttct gttgcaaagt tagaaaaata tagctaacta ccattttatc atgtcgttgt
     3001 tcccttaact ggctaacata tccctaattt cgtggcatgg cgaaaatccg tagatctgaa
     3061 gagatctgcg gttcttttta tatagagcgt aaatacattc aatgcctttg agtgtatttt
     3121 ttgccgtatt gatactttga tatcttgtct ttcttacttt aatatgacgg tgatcttgct
     3181 caatgaggtt attcagatat ttcgatgtac aatgacagtc aggtttaagt ttaaaagctt
     3241 taattacttt agctattgct actttcgttg aaggtgcctg atctgtaatt accttttgag
     3301 gtttaccaaa ttgtttaatg agacgtttga taaacgcata tgctgaatga ttatctcgtt
     3361 gcttacgcaa ccaaatatct aatgtatgtc cctctgcatc aatggcacga tataaatagc
     3421 tccattttcc ttttattttg
//
