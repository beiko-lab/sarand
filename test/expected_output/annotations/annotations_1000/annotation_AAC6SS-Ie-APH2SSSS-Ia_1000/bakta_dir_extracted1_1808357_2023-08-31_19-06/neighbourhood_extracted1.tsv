# Annotated with Bakta
# Software: v1.8.1
# Database: v5.0, light
# DOI: 10.1099/mgen.0.000685
# URL: github.com/oschwengers/bakta
#Sequence Id	Type	Start	Stop	Strand	Locus Tag	Gene	Product	DbXrefs
contig_1	cds	554	1000	+	FKBKAJ_00005		N-acetyltransferase domain-containing protein	SO:0001217, UniRef:UniRef50_Q7WTB9
contig_1	cds	1001	2440	+	FKBKAJ_00010	aac(6')-Ie	bifunctional aminoglycoside N-acetyltransferase AAC(6')-Ie/aminoglycoside O-phosphotransferase APH(2'')-Ia	NCBIProtein:WP_001028144.1, SO:0001217, UniRef:UniRef50_P0A0C2
